apps/module-scan/src/server.ts

```diff
@@ -582,16 +644,42 @@ export async function start({
     workspace = await createWorkspace(workspacePath)
   }
 
-  scanner = scanner ?? new FujitsuScanner({ mode: ScannerMode.Gray })
-  importer =
-    importer ??
-    new Importer({
-      workspace,
-      scanner,
-      workerPoolProvider: (): WorkerPool<workers.Input, workers.Output> =>
-        childProcessPool(workers.workerPath, 2 /* front and back */),
-    })
-  app = app ?? buildApp({ importer, store: workspace.store })
+  scanner ??=
+    process.env.VX_MACHINE_TYPE === 'bsd'
+      ? new FujitsuScanner({ mode: ScannerMode.Gray })
+      : process.env.VX_MACHINE_TYPE === 'precinct-scanner'
+      ? new PlustekScanner({
+          get: memo(
+            (): Promise<Result<ScannerClient, Error>> =>
+              createClient({
+                ...DEFAULT_CONFIG,
+                savepath: workspace!.ballotImagesPath,
+              })
+          ),
+        })
+      : ((): never => {
+          throw new Error(
+            `invalid VX_MACHINE_TYPE=${
+              process.env.VX_MACHINE_TYPE ?? '(blank)'
+            }; expected bsd or precinct-scanner`
+          )
+        })()
```

Comment:
for backwards compatibility, should this default to the fujitsu scanner for now?

Reply:
I'm thinking specifically about making sure that `run-bsd.sh` in `vxsuite-complete-system` works without modification.

Reply:
Sure, can do.